test/e2e/quick_start.go

```diff
@@ -92,6 +93,15 @@ func QuickStartSpec(ctx context.Context, inputGetter func() QuickStartSpecInput)
 			WaitForMachineDeployments:    input.E2EConfig.GetIntervals(specName, "wait-worker-nodes"),
 		})
 
+		By("Upgrading Machine Deployment Kubernetes version to a valid version")
```

Comment:
I think we should keep quick-start job as simple as it is.
Is it possible to have this as part of the upgrade job?

Reply:
+1